openwisp_monitoring/monitoring/migrations/0021_resources_threshold_values.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.0.3 on 2020-08-17 10:18
+
+from django.db import migrations
+
+
+resources_metrics = {'memory': 95, 'disk': 80, 'cpu': 90}
+
+
+def standardize_alertsettings_threshold(apps, schema_editor):
+    AlertSettings = apps.get_model('monitoring', 'AlertSettings')
+    for alert_settings in AlertSettings.objects.all():
```

Comment:
filter only the metrics we're interested in, why looping over all?
filter also the values which are lower than 1